3rdparty/libjasper/jas_stream.c

```diff
@@ -233,6 +233,68 @@ jas_stream_t *jas_stream_memopen(char *buf, int bufsize)
     return stream;
 }
 
+#if defined _WIN32
+jas_stream_t *jas_stream_wfopen(const wchar_t *filename, const char *mode)
+{
+    jas_stream_t *stream;
```

Comment:
Patching of 3rdparty code must be avoided.
Builtin code is a fallback if library binaries missing.
We should support external builds from original sources. Even on Windows, see [vcpkg](https://github.com/Microsoft/vcpkg) project.